gnosis-imap/Makefile

```diff
@@ -38,3 +40,15 @@ SPEC_NAMES:=$(SPEC_NAMES_GNOSIS_SAFE) \
             $(SPEC_NAMES_MODULE_MANAGER)
 
 include ../resources/kprove.mak
+
+# non-standard spec generation
+
+$(SPECS_DIR)/$(SPEC_GROUP)/encodeTransactionData-public-1-spec.k: $(TMPLS) $(SPEC_INI)
+	python3 $(RESOURCES)/gen-spec.py $(TMPLS) $(SPEC_INI) encodeTransactionData-public-1 encodeTransactionData-internal-trusted encodeTransactionData-public-1 > $@
+
+$(SPECS_DIR)/$(SPEC_GROUP)/encodeTransactionData-public-2-spec.k: $(TMPLS) $(SPEC_INI)
+	python3 $(RESOURCES)/gen-spec.py $(TMPLS) $(SPEC_INI) encodeTransactionData-public-2 encodeTransactionData-internal-trusted encodeTransactionData-public-2 > $@
+
+# Identical execution path with execution path of encodeTransactionData-public, so no point to activate it.
+$(SPECS_DIR)/$(SPEC_GROUP)/getTransactionHash-2-spec.k: $(TMPLS) $(SPEC_INI)
+	python3 $(RESOURCES)/gen-spec.py $(TMPLS) $(SPEC_INI) getTransactionHash-2 encodeTransactionData-internal-trusted getTransactionHash-2 > $@
```

Comment:
this is not used now.

Reply:
That's intended. I didn't wanted to activate it because it's useless. But still didn't wanted to delete the spec entirely.

Reply:
well, in that case, you should add the target for both getTransactionHash-1 and getTransactionHash-2.

Reply:
minor: you can avoid the duplication by using the wildcard. that is, replace 1 and 2 by `%` in the target, by `$*` in the receipt.